apps/ui/src/networks/offchain/api/queries.ts

```diff
@@ -366,6 +366,21 @@ export const STRATEGY_QUERY = gql`
   ${STRATEGY_FRAGMENT}
 `;
 
+export const STATEMENTS_AND_USERS_QUERY = gql`
+  query ($statementsWhere: StatementsWhere, $userIds: [String!]) {
+    statements(where: $statementsWhere) {
+      delegate
+      space
+      network
+      statement
+    }
```

Comment:
Maybe as a precaution let's fetch 100 statements there, should be enough to cover members of most spaces, afraid that default (20) might not be always enough.

`first` arg controls how many we fetch.